fastlane/swift/ArgumentProcessor.swift

```diff
@@ -19,68 +19,68 @@ struct ArgumentProcessor {
     let currentLane: String
     let commandTimeout: Int
     let port: UInt32
-    
-    init(args: [String]) {        
+
+    init(args: [String]) {
         // Dump the first arg which is the program name
         let fastlaneArgs = stride(from: 1, to: args.count - 1, by: 2).map {
-            RunnerArgument(name: args[$0], value: args[$0+1])
+            RunnerArgument(name: args[$0], value: args[$0 + 1])
         }
         self.args = fastlaneArgs
-        
+
         let fastlaneArgsMinusLanes = fastlaneArgs.filter { arg in
-            return arg.name.lowercased() != "lane"
+            arg.name.lowercased() != "lane"
```

Comment:
no functional change here?
(I don't know any swift)

Reply:
Let me double-check, but the return is unneeded.

Reply:
Yeah, no `return` keyword is needed in one-line methods like `map`, `flatMap`, `filter`, etc.